lib/fabrik/DirectorManager.js

```diff
@@ -352,6 +341,7 @@ class DirectorManager extends BaseManager {
       name: deploymentName,
       director_uuid: this.director.uuid,
       releases: this.releases,
+      stemcells: [this.stemcell],
       compilation: BoshDirectorClient.getInfrastructure().compilation,
```

Comment:
remove unnecessary things that are not used in Header.js like compilation, disk_pools, resource_pools